src/python/espressomd/particle_data.pyx

```diff
@@ -1678,6 +1678,62 @@ cdef class ParticleList:
                     if (p.type == t or t == "all"):
                         vtk.write("{} {} {}\n".format(*p.v))
 
+
+    def writevsf(self, fp, types='all'):
+        """
+        writes a VST (VTF Structure Format) to a file.
+        This can be used to write the header of a VTF file.
+
+        Parameters
+        ----------
+        types : str
+                Specifies the particle types. The string 'all' will write all particles
+        fp : file
+                   File pointer to write to.
+
+        """
+        global box_l
+        if not hasattr(types, '__iter__'):
+            types = [types]
+            
+        fp.write("unitcell {} {} {}\n".format(*(box_l)))
+        for p in self:
+            for t in types:
+                if (p.type == t or t == "all"):
+                    fp.write("atom {} radius 1 name {} type {} \n".format(p.id, p.type, p.type))
```

Comment:
Just use the LJ radius (I'm not sure whether this is sigma in ESPResSo, there might be internal unit conversions) for the corresponding type of the particle and 1 if there is no active LJ interaction.